Classes/Controller/SeminarController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace OliverKlee\Seminars\Controller;
+
+/**
+ * This file is part of the "seminars" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
```

Comment:
Please let's drop the file doc comments for new classes. (I plan to remove them from the old classes sometime.)